src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -500,7 +482,6 @@ void mongoSubCacheRefresh(const std::string& database)
     }
 
     int r = mongoSubCacheItemInsert(tenant.c_str(), sub);
-
     if (r == 0)
     {
       ++subNo;
```

Comment:
I prefer to defer the decision about this and do it in an independent PR (probably the very last one just before releasing 1.3.0 :))